src/App.js

```diff
@@ -177,41 +158,24 @@ class Board extends React.Component {
     //Holds one true value
     let valueBoard = [];
 
-    //Give value to boards
     for(let i = 0; i < 81; i++) {
```

Comment:
You don't have to make the change in this PR, but I think it is cleaner to use a while loop that doesn't iterate, and explicitly iterate inside based on `position` instead. Then `positionGenerator` can return -1 to move back instead of -2 which is much cleaner.

Reply:
Done!